src/index.js

```diff
@@ -73,6 +75,49 @@ const renderFragment = async (path) => {
   };
 };
 
+/**
+ * Renders content from AEM UE pages
+ */
+const renderContent = async (path, params) => {
+  const { authorization } = params;
+
+  /**
+   * FIXME: Page extension to be handled dynamically. Revisit this implementation and identify a proper
+   * fix.
+   * */
+
+  const aemURL = `${aemConfig.aemEnv}${path}.html?wcmmode=disabled`;
```

Comment:
You should call the franklin delivery servlet which does some important post processing on the html

```
`${aemConfig.aemEnv}/bin/franklin.delivery/${owner}/${repo}/${ref}/${path}?wcmmode=disabled`;
```

AEM will take care of the reverse mapping of the `path`.

Reply:
Addressed